content/kubeflow/install.md

```diff
@@ -21,7 +21,7 @@ eksworkshop-eksctl      ng-52c1fb5e     2019-11-05T16:22:50Z    3
 ```
```

Comment:
Do you think we need to remove first two commands and simplify overall? I think it would be good to remove redundant info. I think we can remove all of the below commands and jump to what you suggest by increasing the nodegroup size
```
eksctl get nodegroups --cluster eksworkshop-eksctl
Take a note of your nodegroup name and use it to scale your cluster. For ex, here is my nodegroup

CLUSTER                 NODEGROUP       CREATED                 MIN SIZE        MAX SIZE        DESIRED CAPACITY    INSTANCE TYPE   IMAGE ID
eksworkshop-eksctl      ng-52c1fb5e     2019-11-05T16:22:50Z    3               3               3          m
Currently my cluster has 3 nodes in ng-52c1fb5e nodegroup. In order to scale my cluster to 6, I use this command
```

Reply:
SGTM.  Done!